tests/console/scc_deregistration.pm

```diff
@@ -0,0 +1,34 @@
+# Copyright (C) 2017 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Deregister from the SUSE Customer Center
+# Maintainer: Qingming Su <qmsu@suse.com>
+
+use strict;
+use base "consoletest";
+use testapi;
+use utils "systemctl";
+use suseconnect_register "suseconnect_deregistration";
+
+sub run {
+    return unless (get_var('SCC_REGISTER') || get_var('HDD_SCC_REGISTERED'));
+
+    select_console 'root-console';
+    suseconnect_deregistration;
+    save_screenshot;
```

Comment:
I wonder if this call is necessary or if the previous call did not already provide a Screenshot